### ISSUE TITLE
Multi-class

### ISSUE THREAD

**Issue by aawink7** [DengPingFan/PraNet#9]
Hi, amazing work!

I want to try training this on my own data,
could you guide me on how to modify the code, the format of the masks, etc. for multi-class segmentation? 
I have to segment out 3 classes + background

**Comment by GewelsJI** [comment]
Hi, aawink7

For multi-class segmentation, you should format your custom dataset in the VOC fashion, and then, modify your `dataloader.py`, finally, modify your loss function

I think existing repos can help you modify the PraNet into multi-class version. Some references: [link-1](https://github.com/zijundeng/pytorch-semantic-segmentation), [link-2](https://github.com/CSAILVision/semantic-segmentation-pytorch)

Or you can refer to ours another project about lung infection segmentation: [Inf-Net](https://github.com/DengPingFan/Inf-Net)

Thank you for your attention.

Best regards,
Ge-Peng.

**Comment by aawink7** [comment]
Thank you so much!
